int32 run_course_status
int32 heartbeat_status
int32 follow_status
int32 docking_status
